test/jasmine/tests/mapbox_test.js

```diff
@@ -292,6 +292,22 @@ describe('mapbox credentials', function() {
                 lon: [10, 20, 30],
                 lat: [10, 20, 30]
             }]);
+        }).toThrow(new Error(constants.missingStyleErrorMsg));
+
+        expect(Lib.error).toHaveBeenCalledTimes(0);
```

Comment:
is this right? `Lib.error` is not called?

Reply:
Correct, we "only" throw an error.

Reply:
... I could change that of course.

Reply:
No changes needed, just ensuring I was reading things right!